### DISCUSSION TITLE
Not able to upload .zwo files

### DISCUSSION THREAD

**Discussion by Animalbmx** [dvmarinoff/Auuki#178] [General]
I created a workout in 'Editor' and downloaded it, I Tried to upload to 'workouts' but I am stuck in a loop of clicking the 'select' button and clicking the file.

Is anyone else experiencing this?

**Comment by dvmarinoff** [comment]
Hi @Animalbmx , thanks for the report!

Here is how it works in Chrome:

https://github.com/dvmarinoff/Flux/assets/7523832/2f849249-6f7a-42ed-9277-7cc641e8e8ed

Can I get more info on your Browser and OS?

**Reply by Animalbmx** [reply]
I'm on Chrome Desktop (Mac) 
Chrome Version 120.0.6099.216

https://github.com/dvmarinoff/Flux/assets/156340808/75489719-b965-4510-ab10-4cdc3d2f36c7

**Reply by dvmarinoff** [reply]
OK, I am on the same setup, so that's not it. My next hypothesis:
1. Maybe the file is causing some issue
2. Maybe the Browser is missing permission to use the Browser storage

Can we try these:
1. send me the file here as attachment
2. make a screenshot of the developer console in the browser.

How to open the console:

Right Click > Inspect > select console tab
 
https://developer.chrome.com/docs/devtools/open

If everything goes right you should see this:
<img width="1432" alt="Screenshot 2024-01-11 at 18 45 34" src="https://github.com/dvmarinoff/Flux/assets/7523832/bb13d1d7-4edb-4dc0-96dd-f488ea91e946">

**Reply by Animalbmx** [reply]
So I made the file a few more times and it happened again, but then I removed the '_' in the file name at it works perfectly.

Thank you for taking the time to respond so quickly and I appreciate the product!

**Reply by dvmarinoff** [reply]
Thanks!

I am still curious why file uploads would fail. I've tried putting '_' (underscore) and ' ' (space) in various places in the file name, but I can't recreate the issue.

**Reply by Animalbmx** [reply]
I think I've found the issue. 
If I try to upload the same file name twice it gets stuck into the loop.
